installation_and_upgrade/check_for_admin_console.bat

```diff
@@ -0,0 +1,10 @@
+REM Prompt user if running console with Administrative privileges.
```

Comment:
This should have a   `setlocal EnableDelayedExpansion` as `!errorLevel!` is used within an if clause, I will push the change